src/Alerts/context/notificationContext.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Should the file name be `NotificationContext` to match how we're doing things elsewhere?

Reply:
sure, wasn't sure if contexts same as hooks start non capital.